packages/fluent-theme/src/fluent/styles/DetailsList.styles.ts

```diff
@@ -41,5 +47,41 @@ export const DetailsRowStyles = (props: IDetailsRowStyleProps): Partial<IDetails
         }
       ]
     ]
+    // TODO: check with designer if needed according to relines.
+    // checkCell: {
+    //   // Move the next cell after check to the left to align to redlines.
+    //   // Temporary using a hard coded value until fluent becomes default and we can access the value from the default constant.
+    //   marginRight: -12
+    // }
+  };
+};
+
+export const DetailsRowCheckStyles = (props: IDetailsRowCheckStyleProps): Partial<IDetailsRowCheckStyles> => {
+  return {
+    check: {
+      // Currently this causes a vertical scroll due to column width calculation logic
+      // in the DetailsList which takes into account the 40px default Check width vs the 48px fluent variation.
+      // When the fluent styles are to become the default ones this will be fixed in the logic.
+      width: detailsRowCheckWidth
+    }
+  };
+};
+
+// Using here any because those specific interfaces are missing from the public API. Consider tightening them in a separate PR.
+// tslint:disable-next-line:no-any
+export const DetailsHeaderStyles = (props: any): any => {
+  return {
+    // TODO: check with designer if needed according to relines.
+    // cellIsCheck: {
+    //   // Move the next cell after check to the left to align to redlines.
+    //   // Temporary using a hard coded value until fluent becomes default and we can access the value from the default constant.
+    //   marginRight: -12
```

Comment:
I'd take the commit / diff approach here to keep the source files clean of commented out code.

Reply:
Sorry... I am not understanding what you mean when you say 

> capture these comments as a diff

I actually talked to Ben yesterday of this new alignment they designed for fluent styles and we came to the conclusion it needs another pass through to make sure we are not breaking anything else. Specifically this part in the toolkit where they don't have a 12px padding between the `Check` and the following cell in the attempt to align with the `GroupHeader`. But this requires changes in the DetailsList columns headers to align everything which I am hesitant to do as it might break some styling in OneDrive. The main issue is that the fluent styles provided by designers fro `DetailsList` and `GroupedList` are targeting mostly their use in OneDrive and not taken into consideration that these are Fabric components that can be used anywhere else. So I will get back to this comments as soon as we have a better approach in styling these parts.

Reply:
My only suggestion was to stash these changes using git, perhaps even by pushing up to a remote branch and linking back in PR or code comment. This would remove the need for or at least the amount of in-line comments (if you want to refer to the commit itself as the comment you can). Then anyone can come thru at a later date, apply the commit, and raise a pull-request as a unit of work.

This is a cleaner approach as the commit will encapsulate the work for a point-in-time. With the comment approach, someone has to figure out what all needs to be changed which a commit does for you.